mu-plugins/blocks/site-breadcrumbs/index.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Block Name: Site Breadcrumbs
+ * Description: Display breadcrumbs of the site.
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\MU_Plugins\Site_Breadcrumbs;
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Registers the block using the metadata loaded from the `block.json` file.
+ * Behind the scenes, it registers also all assets so they can be enqueued
+ * through the block editor in the corresponding context.
+ *
+ * @see https://developer.wordpress.org/reference/functions/register_block_type/
+ */
+function init() {
+	register_block_type(
+		__DIR__ . '/build',
+		array(
+			'render_callback' => __NAMESPACE__ . '\render_block',
+		)
+	);
+}
+
+/**
+ * Render the block content.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the block markup.
+ */
+function render_block( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['postId'] ) ) {
```

Comment:
Can we remove this line? It's causing the control to not show up on the search page.

Reply:
Huh, it's strange that it worked before. In any case, this block uses the global page context, not a local `postId` so that can be removed entirely. Done in https://github.com/WordPress/wporg-mu-plugins/pull/305/commits/730275e166989a137e5283e566bfed5c52bb3b9f.